sample_files/problem_setting/test/run_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+BLUE="\e[0;34m"
+GREEN="\e[0;32m"
+WHITE="\e[0m"
```

Comment:
This isn't white, is it? This is more like "reset".

Reply:
Renamed